src/openforms/authentication/templates/of_authentication/logout_confirmation.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'ui/views/abstract/detail.html' %}
+{% load i18n %}
+
+{% block extra_css %}
+    {% include "forms/sdk_css_snippet.html" %}
+    {{ block.super }} {# Called after SDK CSS for custom design overrides #}
+{% endblock %}
+
+{% block card %}
+    <div class="card">
+        <header class="card__header">
+            <h1 class="title">
```

Comment:
I think we have a `card__title` element?

Reply:
Ah right, the classnames in the SDK are prefixed and these aren't. Let's leave it for another issue/PR to clean that up.